Hi {!GEL_Global_Exception_Log__c.Test_Owner_First_Name__c},

W.r.t. the exception "{!GEL_Global_Exception_Log__c.REF_Number2__c}", further details have been provided by the exception raiser as required, please review and update the Global exceptions log as appropriate.

Exception details are as follows.
Status : {!GEL_Global_Exception_Log__c.Status__c}
REF Number: {!GEL_Global_Exception_Log__c.REF_Number2__c}


Thank you